### PR TITLE
Add missing gio reexports

### LINKED ISSUE #520: BindingBuilder for gio::Settings not exposed in the docs
In the docs it's not possible to find the `BindingBuilder` returned by [`gio::SettingsExtManual::bind()`](https://gtk-rs.org/gtk-rs/git/docs/gio/prelude/trait.SettingsExtManual.html#tymethod.bind). Even rustdocs can't find it.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #520.

cc @jsparber

### CODE REVIEW

File: gio/src/lib.rs [hunk 1]
```diff
@@ -34,13 +34,16 @@ mod file_enumerator;
...
+pub use crate::inet_address::InetAddressBytes;
 mod inet_socket_address;
 mod io_stream;
 pub use crate::io_stream::IOStreamAsyncReadWrite;
 mod input_stream;
 pub use crate::input_stream::{InputStreamAsyncBufRead, InputStreamRead};
 mod list_store;
+#[cfg(test)]
 mod memory_input_stream;
+#[cfg(test)]
```
[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply gio/src/lib.rs:46]: Then we need to add external tests to ensure they are available in those tests. I'm a bit surprised that the tests are in the code directly...

[the remark this answers is not in the document]
Comment (sdroege) [reply gio/src/lib.rs:46]: Where else, that's how we do it everywhere? You'd only put integration tests in a separate executable inside the tests directory. If you'd do that for all the unit tests then the compilation time would go up a lot.

[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply gio/src/lib.rs:46]: Only the compilation time for the tests, not for the crate. And in this case, I'd prefer to have the files containing only tests to be in the `tests` folder instead.

[the remark this answers is not in the document]
Comment (sdroege) [reply gio/src/lib.rs:46]: Yes, make this another PR please and remove the `#[cfg(test)]` so we can merge this now :) Like this currently it's too easy to accidentally add API in there without removing that.

[the remark this answers is not in the document]
Comment (sdroege) [reply gio/src/lib.rs:46]: Apparently we do the same for other modules, so...

**Pull request merged**